src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -306,8 +306,7 @@ public function get($id, $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE
             try {
                 $service = $this->$method();
             } catch (\Exception $e) {
-                unset($this->loading[$id]);
-                unset($this->services[$id]);
+                unset($this->loading[$id], $this->services[$id]);
```

Comment:
So, that pretty live thread here =)
@nicolas-grekas I really like the proposal from @keradus about adding combine_consecutive_unsets into .php_cs.dist. Also previously my PR did already merge almost all unsets over the codebase.